packages/manager/modules/vps/src/dashboard/modal/snapshot-download/index.scss

```diff
@@ -0,0 +1,13 @@
+.download-snapshot-modal {
+  .notification-box-container {
+    min-height: 56px;
+    width: 100%;
+    margin-top: 16px;
+    margin-bottom: 16px;
+  }
+
+  .button-container {
+    display: flex;
+    justify-content: center;
+  }
```

Comment:
**nitpick:** Avoid writing custom css. You can use utilities available from Bootstrap.

Reply:
Done.